collectors/okta/okta_collector.js

```diff
@@ -95,6 +95,9 @@ class OktaCollector extends PawsCollector {
         const ts = parse.getMsgTs(msg, tsPaths);
         const typeId = parse.getMsgTypeId(msg, typeIdPaths);
 
+        // TODO remove this when okta filter this field out
+        delete msg.client.apiToken;
```

Comment:
what happens if `msg.client` property doesn't exist?

Reply:
Delete won't have any effect and just returns `true`

Reply:
The api token is not just in that spot. It needs to be removed in the default http headers.

```
 "apiToken": "WHYUDOTHISOKTA?",
        "http": {
            "defaultHeaders": {
                "Authorization": "SSWS WHYUDOTHISOKTA?",
                "User-Agent": "@okta/okta-sdk-nodejs/3.1.0 node/11.10.0 darwin/18.7.0"
            },
```